recipe/meta.yaml

```diff
@@ -36,7 +36,6 @@ requirements:
     - sqlalchemy
     - beautifulsoup4
     - requests
-    - bsddb  # [unix and py27]
```

Comment:
are we not still supporting 27 here?

Reply:
I wanted to see what this was for.

Reply:
Something needed it on py2 I can't remember what, but I presume it still does...

Reply:
Looks like the tests pass under 2.7 and unix without.